### PR TITLE
[12.0][UPD] openupgrade analysis

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Using https://github.com/OCA/OpenUpgrade/pull/1906 and https://github.com/OCA/OpenUpgrade/pull/1907.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/product/migrations/12.0.1.2/openupgrade_analysis.txt [hunk 1]
```diff
@@ -1,9 +1,11 @@
 ---Fields in module 'product'---
-product      / product.attribute        / attribute_line_ids (one2many) : relation is now 'product.template.attribute.line' ('product.attribute.line')
-product      / product.attribute        / create_variant (boolean)      : now required, default = function
+product      / product.attribute        / attribute_line_ids (one2many) : relation is now 'product.template.attribute.line' ('product.attribute.line') [nothing to do]
+product      / product.attribute        / create_variant (boolean)      : now required
 product      / product.attribute        / create_variant (boolean)      : selection_keys is now '['always', 'dynamic', 'no_variant']' ('False')
 product      / product.attribute        / create_variant (boolean)      : type is now 'selection' ('boolean')
-product      / product.attribute.price  / value_id (many2one)           : DEL relation: product.attribute.value, required: required
+product      / product.attribute.line   / value_ids (many2many)         : table is now 'product_attribute_value_product_template_attribute_line_rel' ('product_attribute_line_product_attribute_value_rel')
```
Comment (MiquelRForgeFlow) [inline addons/product/migrations/12.0.1.2/openupgrade_analysis.txt:6]: works!

File: addons/account/migrations/12.0.1.1/openupgrade_analysis.txt [hunk 2]
```diff
@@ -1,10 +1,11 @@
 ---Fields in module 'account'---
-account      / account.account          / internal_group (selection)    : NEW selection_keys: function
+account      / account.account          / internal_group (selection)    : NEW isrelated: related, stored: stored, selection_keys: function
+account      / account.account.tag      / color (integer)               : without default
```
Comment (pedrobaeza) [inline addons/account/migrations/12.0.1.1/openupgrade_analysis.txt]: This text disappears now, isn't it?

Comment (MiquelRForgeFlow) [reply addons/account/migrations/12.0.1.1/openupgrade_analysis.txt]: yes, don't merge this yet, I need to redo the analysis

Comment (pedrobaeza) [reply addons/account/migrations/12.0.1.1/openupgrade_analysis.txt]: Yeah, I was waiting for that :smiley:

**Comment by MiquelRForgeFlow** [comment]
updated

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please run again the analysis with latest improvements avoiding duplicated words.

**Comment by MiquelRForgeFlow** [comment]
updated!! :green_apple:

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**